modules/createHashHistory.js

```diff
@@ -133,7 +133,7 @@ function createHashHistory(options={}) {
 
   function listenBefore(listener) {
     if (++listenerCount === 1)
-      stopHashChangeListener = startHashChangeListener(history)
```

Comment:
The changes are only needed for tests, to use a spy to check of transitionTo is called.